src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -686,7 +686,10 @@ public void setAllowEmptyCatches(boolean allow) {
     @Override
     public void visitToken(DetailAST ast) {
         final int currentType = ast.getType();
-        if (!isNotRelevantSituation(ast, currentType)) {
+        int childType = 0;
+        if(ast.hasChildren()){
+            childType = ast.getLastChild().getType();}
+        if (!isNotRelevantSituation(ast, currentType) || isArrayInitialization(currentType, childType) ){
```

Comment:
@romani @rnveach I am facing a bit of trouble over here. 
Other test cases and their files have arrays in them, that obviously don't have spacing according to the test, so those tests are also failing, so this will mean to modify all the resource files (Modify arrays in them so that test case passes)  in `whitespacearound`. Is there any other solution?

Reply:
And the checks failing below are because of the unit tests failing for the reason mentioned above.